### PR TITLE
Update module mapping with `IdentityProtection` singleton

### LINKED ISSUE #602: Get-MgRiskDetection and Get-MgRiskyUser are only available with beta profile
Both of these endpoints were promoted to v1.0 back in May 2020. Why are the cmdlets only available when using beta? 
[AB#8809](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8809)

### PR DESCRIPTION
**Pull request by peombwa**
This PR closes #602 by replacing `riskyUsers` and `riskDetections` with `IdentityProtection` singleton in module mapping.  `riskyUsers` and `riskDetections` APIs were moved  `IdentityProtection` singleton.

See https://docs.microsoft.com/en-us/graph/api/riskyusers-list?view=graph-rest-beta&tabs=http#http-request for more details.

### CODE REVIEW

**Comment by peombwa** [comment]
This PR is currently blocked by https://github.com/microsoft/OpenAPI.NET.OData/issues/221 in DevX API. DevX API needs to be updated with https://github.com/microsoft/OpenAPI.NET.OData/pull/226 before the PR can be merged.

**Review by timayabi2020 — DISMISSED** [review]

**Review by timayabi2020 — APPROVED** [review]

**Pull request merged**
